terra/contracts/token-bridge/src/contract.rs

```diff
@@ -121,8 +121,41 @@ const CHAIN_ID: u16 = 3;
 const WRAPPED_ASSET_UPDATING: &str = "updating";
 
 #[cfg_attr(not(feature = "library"), entry_point)]
-pub fn migrate(_deps: DepsMut, _env: Env, _msg: MigrateMsg) -> StdResult<Response> {
-    Ok(Response::new())
+pub fn migrate(deps: DepsMut, env: Env, _msg: MigrateMsg) -> StdResult<Response> {
+    let messages = migrate_wrapped_assets(deps, env, 767)?;
```

Comment:
Where is this number coming from?  Is there some meaning behind it or is it just random?  What should be the next `code_id`?  When should this be removed?

Please add comments in the code explaining this.

Reply:
Added an explanation of this code.